mat64/dense_arithmetic.go

```diff
@@ -49,6 +54,10 @@ const (
 	small   = math.SmallestNonzeroFloat64
 )
 
+// Norm returns the specified matrix p-norm of the matrix represented by the receiver.
+// It currently supports p = 1, -1, +Inf, -Inf, 2, -2.
```

Comment:
Remove this line. While it's marginally helpful, the "-inf" norm is not clear. All this documentation is in Normer, so we should just direct the user there directly.